### DISCUSSION TITLE
Problem running editor, cannot open editor.ui

### DISCUSSION THREAD

**Discussion by ghost** [AnthonyMichaelTDM/DRG-Save-Editor#45] [Q&A] [answered]
![image](https://github.com/AnthonyMichaelTDM/DRG-Save-Editor/assets/56131929/c5bcb620-bf57-43d7-b9ff-9620a6c3fa68)
The german text means "System cannot find given file"

**Comment by ghost** [comment]
after doing some testing, it seems like its able to run if i move both definitions.py and main.py into the root folder

**Comment by AnthonyMichaelTDM** [comment, marked as answer]
I guess that's one way to do it but, the script is intended to be executed from the project root as per the readme
